### PR TITLE
Allow running runtime_unittests in AOT mode.

### PR DESCRIPTION
**Pull request by chinmaygarde**
Previously, only the most basic tests were run in AOT mode.

### CODE REVIEW

**Comment by jason-simmons** [comment]
Looks like the debug version of this test is now failing on Cirrus?

File: runtime/dart_isolate_unittests.cc [hunk 1]
```diff
@@ -130,7 +127,7 @@ class AutoIsolateShutdown {
   FML_DISALLOW_COPY_AND_ASSIGN(AutoIsolateShutdown);
 };
 
-std::unique_ptr<AutoIsolateShutdown> RunDartCodeInIsolate(
+static std::unique_ptr<AutoIsolateShutdown> RunDartCodeInIsolate(
```
Comment (GaryQian) [inline runtime/dart_isolate_unittests.cc]: Is this meant to be static?

Comment (chinmaygarde) [reply runtime/dart_isolate_unittests.cc]: Yes. This is just a utility method.

**Comment by chinmaygarde** [comment]
Fixed the debug variant.

**Pull request merged**